stanford/bin/audit-iam-users

```diff
@@ -144,10 +143,46 @@ def print_items(users):
         print('- ' + user['user'])
 
 
+def audit_groups_empty(client, users):
+    print('## empty groups')
+    print('### TODO: delete group')
+    groups = get_groups_empty(client, users)
+    groups = sorted(groups)
+    for group in groups:
+        print('- ' + group)
+    print('')
+
+
+def get_groups_empty(client, users):
+    groups = get_groups(client)
+    for user in users:
+        username = user['user']
+        user_groups = get_groups(client, username)
+        groups = groups - user_groups
+    return groups
+
+
+def get_groups(client, username=None):
```

Comment:
It seems like renaming this get_all_groups would make it clearer.

Reply:
Sounds good; done.